<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tasks</class>
 <widget class="QWidget" name="Tasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="allProjectsCheckBox">
       <property name="text">
        <string>Show All Projects</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showApprovedCheckBox">
       <property name="text">
        <string>Show Done/Blocked</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tasksTableWidget">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tasksTableWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Tasks</receiver>
   <slot>setCurrentTask(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showApprovedCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Tasks</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allProjectsCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Tasks</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCurrentTask(QModelIndex)</slot>
  <slot>updateFilters()</slot>
 </slots>
</ui>
